JavaDoc errors in ThreadContext and ContextManager found by jenkins build

Having made it past the previous error, I've now run into the following JavaDoc errors when running the jenkins build.
Fortunately, these are easy to reproduce locally and should be simple to fix,
```
[INFO] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (attach-javadocs) on project microprofile-context-propagation-api: MavenReportException: Error while generating Javadoc: 
[INFO] [ERROR] Exit code: 1 - /home/jenkins/agent/workspace/MicroProfile Releases/api/src/main/java/org/eclipse/microprofile/context/ThreadContext.java:318: error: unexpected end tag: </p>
[INFO] [ERROR]      * </p>
[INFO] [ERROR]        ^
[INFO] [ERROR] /home/jenkins/agent/workspace/MicroProfile Releases/api/src/main/java/org/eclipse/microprofile/context/spi/ContextManager.java:96: error: reference not found
[INFO] [ERROR]          * {@link CompletableFuture} and {@link CompletionStage} when no executor is specified.
[INFO] [ERROR]                   ^
[INFO] [ERROR] /home/jenkins/agent/workspace/MicroProfile Releases/api/src/main/java/org/eclipse/microprofile/context/spi/ContextManager.java:96: error: reference not found
[INFO] [ERROR]          * {@link CompletableFuture} and {@link CompletionStage} when no executor is specified.
[INFO] [ERROR]                                                 ^
```

--- a/api/src/main/java/org/eclipse/microprofile/context/ThreadContext.java
+++ b/api/src/main/java/org/eclipse/microprofile/context/ThreadContext.java
@@ -315,7 +315,6 @@ public interface ThreadContext {
      * <pre>mp.context.ThreadContext.unchanged=None</pre>
      * or
      * <pre>mp.context.ManagedExecutor.propagated=None</pre>
-     * </p>
      */
     static final String[] NONE = new String[0];
 

--- a/api/src/main/java/org/eclipse/microprofile/context/spi/ContextManager.java
+++ b/api/src/main/java/org/eclipse/microprofile/context/spi/ContextManager.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018,2019 Contributors to the Eclipse Foundation
+ * Copyright (c) 2018,2020 Contributors to the Eclipse Foundation
  *
  * See the NOTICE file(s) distributed with this work for additional
  * information regarding copyright ownership.
@@ -18,6 +18,8 @@
  */
 package org.eclipse.microprofile.context.spi;
 
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
 import java.util.concurrent.ExecutorService;
 
 import org.eclipse.microprofile.context.ManagedExecutor;

